### DISCUSSION TITLE
How can I implement/override structure of the shops

### DISCUSSION THREAD

**Discussion by deiby-dayans** [osiset/laravel-shopify#906] [Q&A]
Hi,

I'm trying to build a feature that includes some migrations, and new data related to the shops.

I want to know how to catch the creation of the Shop/User record after the authorization and then include my own functionality that includes save some default data in new tables related with the each of the new shop.

Thanks,

Deiby

**Comment by gnikyt** [comment]
You can use observers to know when a shop is created. Look up Laravel observers, you can tie one to on-create of a new user (shop).
